test/module.mk

```diff
@@ -6,7 +6,14 @@
 ######################################################################
 
 TESTS        := $(srcdir)/test/common/*.h $(srcdir)/test/audio/*.h $(srcdir)/test/math/*.h
-TEST_LIBS    := audio/libaudio.a math/libmath.a common/libcommon.a
+TEST_LIBS    := test/null_osystem.o \
+	backends/fs/posix/posix-fs-factory.o \
+	backends/fs/posix/posix-fs.o \
+	backends/fs/posix/posix-iostream.o \
+	backends/fs/stdiostream.o \
+	backends/modular-backend.o \
+	test/stubs.o \
```

Comment:
Don't you need to add backends/mixer/null/null-mixer.o here? Or else to add stubs for the `NullMixerManager` functions?

Reply:
I tried it now, and I indeed get undefined symbols:
```
Undefined symbols for architecture x86_64:
  "NullMixerManager::update(unsigned char)", referenced from:
      OSystem_NULL::pollEvent(Common::Event&) in null_osystem.o
  "NullMixerManager::NullMixerManager()", referenced from:
      OSystem_NULL::initBackend() in null_osystem.o
```
Adding `backends/mixer/null/null-mixer.o` to test/module.mk fixed the issue.

Note: I also got another compilation error
```
test/../backends/platform/null/null.cpp:118:8: error: unknown type name 'sighandler_t'
static sighandler_t last_handler;
       ^
```
That one is because sighandler_t is a GNU extension and I am on macOS. I fixed it by adding the following after including `signal.h` in backends/platform/null/null.cpp. This does not necessarily need to be part of this PR though, and I can commit the change separately.
```
// sighandler_t is a GNU extension exposed when _GNU_SOURCE is defined
#ifndef _GNU_SOURCE
typedef void (*sighandler_t)(int);
#endif
```